rpcserver.go

```diff
@@ -1460,7 +1460,31 @@ func (r *rpcServer) SendPayment(paymentStream lnrpc.Lightning_SendPaymentServer)
 					// TODO(roasbeef): eliminate necessary
 					// encode/decode
 					nextPayment.Dest = payReq.Destination.SerializeCompressed()
-					nextPayment.Amt = int64(payReq.Amount)
+					if payReq.MilliSat == nil {
+						err := fmt.Errorf("only payment" +
+							" requests specifying" +
+							" the amount are" +
+							" currently supported")
+						select {
+						case errChan <- err:
+						case <-reqQuit:
+							return
+						}
+						return
+					}
+					nextPayment.Amt = int64(payReq.MilliSat.ToSatoshis())
```

Comment:
This is incorrect. All payment amounts should be handled as `mSAT` internally.

Reply:
You're right. Changed to always sending msat over `payChan`